app/flows/check_benefits_financial_support_flow/outcomes/results.erb

```diff
@@ -34,9 +34,8 @@
     <%= render "govuk_publishing_components/components/list", {
     visible_counters: true,
     items: [
-      sanitize("browsing <a class=\"govuk-link\" href=\"/browse/benefits\">information on benefits</a>"),
-      sanitize("reading about <a class=\"govuk-link\" href=\"https://costoflivingsupport.campaign.gov.uk/\">what the government is doing to support people with the cost of living</a>"),
-      sanitize("<a class=\"govuk-link\" href=\"/benefits-calculators\">using a benefits calculator</a>"),
+      sanitize("reading about <a class=\"govuk-link\" href=\"/cost-of-living\">the support available to help with the cost of living</a>"),
+      sanitize("<a class=\"govuk-link\" href=\"/benefits-calculators\">using a benefits calculator</a> - enter information on income, savings and spending to get an estimate of benefits you can get and how much"),
```

Comment:
Nit, since it's a change not introduced here. 

We're not benefiting from using sanitize here, it's also rather weird that we'd want to sanitize the HTML we've hard coded.